BeagleBone_black/src/state_machine/main.cpp

```diff
@@ -51,7 +51,9 @@ void Main::run()
   while (1) {
     checkFailure();
```

Comment:
update your data, do you ever refresh your data? use `data_.getNavigationData()` and `data_.getCommData()`